content/200-concepts/100-components/02-prisma-client/052-advanced-type-safety/index.mdx

```diff
@@ -127,4 +127,109 @@ userPosts.posts
 userPosts.profile
 ```
 
-> See the [model query options](/reference/api-reference/prisma-client-reference#model-query-options) reference for more information about the different types available.
+> See the [model query options](../../../../reference/api-reference/prisma-client-reference#model-query-options) reference for more information about the different types available.
+
+### Generated <inlinecode>UncheckedInput</inlinecode> types
+
+The `UncheckedInput` types are a special set of generated types that allow you to perform some operations that Prisma Client considers "unsafe", like directly writing foreign keys. You can choose either the "safe" `Input` types or the "unsafe" `UncheckedInput` type when doing operations like `create`, `update`, or `upsert`.
+
+The first tab below shows an example Prisma schema with a one-to-many relation between `User` and `Post`. The second tab shows the `PostUncheckedCreateInput` generated type. It contains the `authorId` property, which is a foreign key. The third tab shows an example query that uses the `PostUncheckedCreateInput` type. This query will result in an error if a user with an `id` of `1` does not exist.
```

Comment:
I think the page would be easier to read if the first tab would be split out to stand by itself, and then the two tabs each for the Unchecked and Checked inputs beneath that.